libs/sdk-backend-tiger/src/convertors/toBackend/afm/MeasureConverter.ts

```diff
@@ -62,37 +72,39 @@ function convertMeasureDefinition(definition: IMeasureDefinitionType): ExecuteAF
 
 function convertAggregation(
     aggregation?: MeasureAggregation,
-): ExecuteAFM.SimpleMeasureAggregation | undefined {
+): SimpleMeasureDefinitionMeasureAggregationEnum | undefined {
     if (!aggregation) {
         return undefined;
     }
     if (aggregation === "sum") {
-        return "SUM";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.SUM;
     }
     if (aggregation === "avg") {
-        return "AVG";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.AVG;
     }
     if (aggregation === "count") {
-        return "COUNT";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.COUNT;
     }
     if (aggregation === "max") {
-        return "MAX";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.MAX;
     }
     if (aggregation === "median") {
-        return "MEDIAN";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.MEDIAN;
     }
     if (aggregation === "min") {
-        return "MIN";
+        return SimpleMeasureDefinitionMeasureAggregationEnum.MIN;
     }
 
-    return "RUNSUM";
+    return SimpleMeasureDefinitionMeasureAggregationEnum.RUNSUM;
 }
 
-function convertSimpleMeasureDefinition(definition: IMeasureDefinition): ExecuteAFM.ISimpleMeasureDefinition {
+function convertSimpleMeasureDefinition(definition: IMeasureDefinition): SimpleMeasureDefinition {
     const { measureDefinition } = definition;
 
-    const filters: ExecuteAFM.FilterItem[] = measureDefinition.filters
-        ? compact(measureDefinition.filters.map(convertVisualizationObjectFilter))
+    const filters: FilterDefinitionForSimpleMeasure[] = measureDefinition.filters
+        ? (compact(
+              measureDefinition.filters.map(convertVisualizationObjectFilter),
+          ) as FilterDefinitionForSimpleMeasure[]) // TODO je to správně?
```

Comment:
Why the cast? What is the inferred type?

Reply:
commented in latest version